src/FingersCrossedProcessor.php

```diff
@@ -3,12 +3,14 @@
 
 namespace League\Pipeline;
 
-class FingersCrossedProcessor implements ProcessorInterface
+class FingersCrossedProcessor implements ProcessorInterface, ParametersInterface
 {
+    use ParametersTrait;
+
     public function process($payload, callable ...$stages)
```

Comment:
Hmm I forgot about this signature, makes it significantly trickier to handle. It's pretty straight forward for the Pipeline:

```php
function process(...$parameters) { ... }
```

But the processor is different because it receives the stages:

```php
function process($payload, callable ...$stages) { ... }
```